src/trans-netlist/trans_trace.cpp

```diff
@@ -74,6 +74,28 @@ unsigned trans_tracet::get_min_failing_timeframe() const
 
 /*******************************************************************\
 
+Function: bvrep2binary
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+std::string bvrep2binary(const constant_exprt &expr)
+{
+  const auto &value = expr.get_value();
+  std::size_t width=to_bitvector_type(expr.type()).get_width();
+  std::string result(width, ' ');
+  for(std::size_t index = 0; index<width; index++)
+    result[width-index-1] = get_bvrep_bit(value, width, index) ? '1' : '0';
+  return result;
+}
```

Comment:
That was my first idea -- but recall that `bv_typet` doesn't have a numerical interpretation.

Reply:
Good point. I still find the proposed code a bit confusing, because there never actually will be any whitespace in the final string. So how about:
```
std::string result;
result.reserve(width);
for(std::size_t dest_index = 0; dest_index < width; ++dest_index)
  result.push_back(get_bvrep_bit(value, width, width - dest_index - 1) ? '1' : '0');
```

Reply:
I am now doing the above for signedbv and unsignedbv.